src/components/atoms/Timeline/index.tsx

```diff
@@ -159,23 +204,31 @@ const MemoryList: React.FC<MemoryListProps> = memo(
           }
           return <Memory key={`${idx}`} isHour={isHour} isStrongMemory={isStrongMemory} />;
```

Comment:
This component is rendered so many times, so this may make performance degrade.
When we need to improve performance we can try [this section in emotion](https://emotion.sh/docs/performance).
But this will make readability degrade.